src/main/java/io/github/terra121/projection/GeographicProjection.java

```diff
@@ -277,4 +277,42 @@ public float azimuth(double x, double y, float angle) throws OutOfProjectionBoun
 	public enum Orientation {
 		none, upright, swapped
 	}
+	public enum Facing {
+		N (337.5, 360, "North"),
+		N2 (0, 22.5, "North"),
```

Comment:
There is no reason to distinguish two different zones for north, this will just make things more complicated.